klayout/drc/rule_decks/mcell.drc

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
+  #================================================
+  #---------------------MCELL----------------------
+  #================================================
```

Comment:
same comment at #57, it would be nice to link to https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_07_18.html#mcell but non blocking.